app/src/main/java/org/mozilla/tv/firefox/channels/content/SportsChannels.kt

```diff
@@ -24,13 +24,5 @@ fun ChannelContent.getSportsChannels(): List<ChannelTile> = listOf(
         setImage = setImage(R.drawable.tile_sports_formula_1),
         tileSource = TileSource.SPORTS,
         id = "formula1"
-    ),
-    ChannelTile(
-        url = "https://www.cbssports.com/videos/",
-        title = "CBS Sports",
-        subtitle = null,
-        setImage = setImage(R.drawable.tile_sports_cbs_sports),
```

Comment:
It might be good to remove this image so that, if we never add CBS sports back again, we're not bundling and shipping that asset forever.

Reply:
I believe it will be Proguarded out, but it doesn't hurt to remove it.  Easy to put back in.